Rewrite install section

gassmoeller opened the pull request:
This is only a first try to address #1818.
I am open for suggestions about which parts need to be more verbose. I would just like to avoid duplicating documentation between candi and the ASPECT manual, because they will get out of sync eventually.

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+In its default configuration it downloads and
+compiles a \dealii{} configuration that is able to run \aspect, but it also contains a number of
+packages that are not required (and that can be safely disabled if problems occur during the
+installation). We require at least the packages \pfrst{}, one of either \trilinos{} or \texttt{petsc},
```
maybe say "..., \trilinos{} (or alternatively, if you want to experiment, \texttt{PETSc})" to make clear that almost everyone is actually using the former.

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+installation). We require at least the packages \pfrst{}, one of either \trilinos{} or \texttt{petsc},
+and finally \dealii{}.
+ 
+At the time of this writing \texttt{candi} will install \pfrst{} 2.0, \trilinos{} 12.10.1,  \texttt{petsc} 3.6.4, and \dealii{} 8.5.0. We strive to keep the development version of \aspect{} compatible to the latest release of deal.II and the current deal.II development version at any time, and we usually support several older versions of \pfrst{}, \trilinos{}, and \texttt{petsc}.
```
-> PETSc

bangerth replied:
compatible to -> compatible with
deal.II -> \dealii{} (twice)

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+    cd candi
+    ./candi.sh -p INSTALL_PATH
+    \end{verbatim} 
+    (here we assume you replace INSTALL\_PATH by the path were
```
-> \texttt{INSTALL\_PATH}

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+    ./candi.sh -p INSTALL_PATH
+    \end{verbatim} 
+    (here we assume you replace INSTALL\_PATH by the path were
+    you want to install all dependencies and \dealii{}). 
```
Maybe add "A typical directory for this would be \texttt{$HOME/bin} or a similar place somewhere under your home directory."

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+    \end{verbatim} 
+    (here we assume you replace INSTALL\_PATH by the path were
+    you want to install all dependencies and \dealii{}). 
+    This step might take a long time, but can be parallelized by adding verbatim{-jx}, where 
```
-> \verbatim?

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
+\item You may now want to configure your environment to make it aware of the newly installed
+    packages. This can be achieved by adding the line 
+    \texttt{source INSTALL\_PATH/configuration/enable.sh} to the file responsible for setting
+    up your shell environment \footnote{For bash this would be the file \tt{\~{}/.bashrc}.} 
```
no space before the `\footnote`

I think you need to say `\texttt`

bangerth commented on doc/manual/manual.tex:
```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
-    this would be adding the line {\tt{export DEAL\_II\_DIR=/path/to/deal-installed/}} to
-    the file {\tt{\~{}/.bashrc}}. Then close the terminal and open it again to activate the change.}
-{\tt{DEAL\_II\_DIR}} to the directory where you \textit{installed} \dealii.
+  by compiling the {\tt{step-32}} example that you can find in
```
same here and below

gassmoeller commented:
I addressed the comments, reinserted the system prerequisites subsection and updated a few minor things in the compile section. Ready to merge when the tester is finished.
